src/components/Configurator/Configurator.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { OptionList, OptionItem, OptionLeft, OptionRadio } from "./styled";
+import { priceParser } from "helpers/cartHelper";
+
+const Configurator = ({ variants, selectedVariant, onChange }) => {
+  if (variants.length === 1) {
+    return null;
+  }
+
+  return (
+    <OptionList>
+      {variants.map(variant => (
+        <OptionItem
+          key={variant.code}
+          selected={selectedVariant.code === variant.code}
+          onClick={() => onChange(variant)}
+        >
+          {variant.axis.join(" | ")}
+          <OptionLeft>
+            {priceParser(variant.price.current, variant.price.currency)}
+            <OptionRadio
+              id={variant.code}
+              type="radio"
+              value={variant.code}
+              checked={selectedVariant.code === variant.code}
+            />
+          </OptionLeft>
+        </OptionItem>
+      ))}
+    </OptionList>
+  );
+};
+
+Configurator.propTypes = {
+  variants: PropTypes.array.isRequired,
```

Comment:
Variants propTypes can be more exhaustive :  
```suggestion
  variants: PropTypes.arrayOf(PropTypes.shape({
    code: PropTypes.string.isRequired,
    price: Proptypes.shape({
      current: PropTypes.string.isRequired,
      currency: PropTypes.string.isRequired
    }).isRequired
  })).isRequired,
```

(I'm no really sure of my proposition code)

Reply:
Thanks, I'll update it on my side to test the shape validity